docs/development/documentation_preview.rst

```diff
@@ -4,13 +4,20 @@
 Documentation Preview
 *********************
 
-Most of the time it's enough to build the documentation locally and see how things are going. But sometimes 
-it's nice to share our changes and get some feedback from other collaborators. 
+To preview your changes to the documentation please:
 
-To preview your changes to the documentation first make sure you have GitHub Actions activated on your fork
-(see the Actions tab). Then, there are two ways to trigger the build:
+#. Enable GitHub Actions to run on your fork (see the *Actions* tab).
+#. Make sure your fork is deploying GitHub Pages inside the root of ``gh-pages`` branch (go to *Settings* -> *Pages*).
+
+Then, there are two ways to trigger the build:
+
+#. If the branch you are working on contains the word ``doc`` in it, then every commit pushed to that branch will trigger the build.
+#. If your commit message contains the ``[build docs]`` tag, then that commit will trigger the build.
+
+.. note::
+
+    If you forgot to folow any of the two ways described above, you always can trigger push an empty commit
```

Comment:
can we make this

you always can trigger push an empty commit in the command line as follows: